### PR TITLE
Snacbar and Grid Added to the component library.

### PR DESCRIPTION
**Pull request by ashifkhn**
Inspect the deploy log: https://app.netlify.com/sites/whyui/deploys/621238749a4b220008cb5a0c

Browse the preview: https://deploy-preview-12--whyui.netlify.app/

### CODE REVIEW

File: components/Grid/Grid.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+.container {
+  display: flex;
+}
+
+.sidebar {
+  display: flex;
+  flex-direction: column;
+  width: 250px;
```
Comment (harshitpaliwal95) [inline components/Grid/Grid.css:17]: use rems as units

**Review by harshitpaliwal95 — COMMENTED** [review]
- overall looks good 
- one suggestion grid direction should be column

**Review by Pratham82 — COMMENTED** [review]
1. If possible add items in row as for grid component.
Everything else looks good.

File: components/Grid/Grid.css (hunk 1 above)
Comment (ashifkhn) [reply components/Grid/Grid.css:17]: Done.

**Pull request merged**